packages/@okta/vuepress-site/docs/guides/implement-auth-code/setup-app/index.md

```diff
@@ -4,9 +4,9 @@ title: Set up your Application
 
 You set up your OpenID Connect application inside the Okta Admin Console:
 
-1. From the side navigation, select **Applications** and then **Applications**.
-1. On the Applications page, choose **Add Application**.
-1. On the Create New Application page, select **Web**.
-1. Fill in the **Application Settings**, then click **Done**. The **Login redirect URIs** must match a URI that your user can be redirected to with their authorization code.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Create App Integration**.
+1. On the Create a new app integration page, select **OIDC - OpenID Connect** for the **Sign-in method** and select **Web Application** as the **Application type**.
```

Comment:
```suggestion
1. On the Create a new app integration page, select **OIDC - OpenID Connect** for the **Sign-in method**, and select **Web Application** as the **Application type**.
```

Reply:
This is actually a two part modal - so I added "and then select" instead